api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/TxMlDataset25.java

```diff
@@ -130,7 +130,21 @@ public DataSetDefinition constructtxMlDataset() {
                     mappings)),
             mappings),
         getColumnsForAgeAndGenderAndKeyPop());
-    // Refused Or Stopped Treatment
+
+    //    // Totals Refused Or Stopped Treatment
```

Comment:
This has been added but commented. If not needed anymore, it can be removed @TinoMuianga

Reply:
The implementation of this feature was carried out to accommodate an additional disaggregation known as "Refused Or Stopped Treatment for TX ML." However, it is not intended for use in the current release. This is the reason why I have retained it in the code as a commented section, reserved for potential future use. Although it was already commented on previously, just not with the updated implementation, which has now been included for clarity and reference.

Reply:
This is clear and makes sense. Thank you @TinoMuianga